keycloak/keycloak-admin-ui
# added unlock user when brute force detect on

+ some light refactor


## Motivation
implements point E on this [marvel](https://marvelapp.com/prototype/7i7ja65/screen/75693215)

## Verification Steps

1. Trurn on brute force detection in realm settings
1. Get a user to be locked out after failed login
1. Go to the users details and see the new Switch
1. Use it to unlock the user

## Checklist:


## Additional Notes
